.github/workflows/python-tests.yml

```
@@ -97,13 +121,23 @@ jobs:
       # circuits and run in their own workflow so that a hiccup at github.com
-      # cannot redden an unrelated matrix.
-      - name: ⬇️ Test the lowest supported dependency versions
+      # cannot redden an unrelated matrix. Skipped on windows-11-arm, see above.
+      - if: inputs.runs-on != 'windows-11-arm'
+        name: ⬇️ Test the lowest supported dependency versions
         run: uvx nox -s minimums --verbose -- --full -m "not network"
 
-      - name: 🐍 Test
+      - if: inputs.runs-on == 'windows-11-arm'
+        name: ⬇️ Test the lowest supported dependency versions (Windows ARM64)
+        run: uvx nox -s minimums ${NOX_PYTHON_MINIMUMS} --verbose -- -m "not network and not torch"
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:21: Double quote to prevent globbing and word splitting

(shellcheck)